lib/flame.dart

```diff
@@ -23,6 +24,13 @@ class Flame {
   /// Access a shared instance of the [FlameAudio] class.
   static FlameAudio audio = FlameAudio();
 
+  /// Access a shared instance of the [Bgm] class.
+  static Bgm _bgm;
+  static Bgm get bgm {
```

Comment:
I might be wrong (please test carefully) but I think you can return the assignament and then you can just d
`static Bgm get bgm => _gm ??= Bgm();`

Reply:
checking it out

Reply:
i tested it and it seems to work:
```dart
int asdf;
int get aassddff => (asdf ??= 5);

print(asdf);
print('------------');
print(aassddff);
print('------------');
print(asdf);
```

got (as expected):
```dart
I/flutter ( 4439): null
I/flutter ( 4439): ------------
I/flutter ( 4439): 5
I/flutter ( 4439): ------------
I/flutter ( 4439): 5
```

any problem with multi-line getter functions though?

Reply:
absolutely no problem, but if we can make any code more concise and legible, I prefer that :) but nothing specific to multi-line getter functions